samples/sensor/sht3xd/src/main.c

```diff
@@ -50,6 +50,10 @@ void main(void)
 	}
 	if (rc == 0) {
 		rc = sensor_trigger_set(dev, &trig, trigger_handler);
+		if (rc != 0) {
+			printf("Failed to set trigger: %d\n", rc);
+			return;
+		}
 	}
```

Comment:
Shouldn't the check be put here to catch also the errors of the previous API (sensor_attr_set called twice)?

Reply:
Yes, the intent of the control flow as to check `rc == 0` after the chain of tests, and fail there, as is done in the similar chain within the while loop.